lib/engine/game/g_1888/step/special_track.rb

```diff
@@ -27,37 +27,6 @@ def legal_tile_rotation?(entity, hex, tile)
 
             tile.rotation.zero?
           end
-
-          # needed to deal with non-plain track connections
-          #
-          def check_connect(_action, ability)
```

Comment:
out of curiosity how is it possible you deleted this

Reply:
The base method works fine now. Before, it would work with non-plain tiles.